app/controllers/candidate_interface/apply_from_find_controller.rb

```diff
@@ -1,6 +1,6 @@
 module CandidateInterface
   # The Apply from Find page is the landing page for candidates coming from the
-  # Find postgraduate teacher training (https://www.find-postgraduate-teacher-training.service.gov.uk/)
+  # Find teacher training courses (https://www.find-postgraduate-teacher-training.service.gov.uk/)
```

Comment:
Is the url wrong here?

Reply:
Yes! But the url and domains will be done in a separate PR.